packages/react-baseline-inputs/src/RadioGroup.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { Field } from "./Field";
+import { FieldSet } from "./FieldSet";
+import { RadioGroupProps } from "./types";
+
+/**
+ * A list of HTML `<input type="radio" />` inputs,
+ * grouped inside a `<fieldset>` and labelled with a
+ * `<legend>`
+ */
+export const RadioGroup: React.FC<RadioGroupProps> = ({
+  title,
+  name,
+  options,
+  inline,
+  value,
+  onChange,
+  ...props
+}) => {
+  const handleChange = React.useCallback(
+    event => onChange(event.target.value || null),
```

Comment:
The value prop for an option is required, so we should avoid defaulting this to `null`

```suggestion
    event => onChange(event.target.value),
```